dep_check/infra/python_parser.py

```diff
@@ -89,6 +89,7 @@ def wildcard_to_regex(self, module: ModuleWildcard) -> str:
         """
         module_regex = module.replace(".", "\\.").replace("*", ".*")
         module_regex = module_regex.replace("[!", "[^").replace("?", ".?")
+        module_regex = module_regex.replace("(<", "(?P<")
```

Comment:
I'd skip this, I'm starting to think we should probably have gone full regular expressions in the first place.
```suggestion
```

Reply:
I try to do so, but it fails due to the `.replace("?", ".?")` just above

Maybe in future, we should deprecate our custom wildcard and rollback to bare regex with a trick for `.`